journal/updated/02-chapter-1.adoc

```diff
@@ -201,65 +204,66 @@ supposing you might be at the Quarterly Meeting.
 I hope you will not be negligent in attending such meetings,
 nor any other which are appointed for the service of Truth,
 and for the worship of Almighty God.
-I pray you, let not the indifferency of others lull you asleep;
+I beseech you, let not the indifferency of others lull you asleep,
 but be diligent and watchful, truly waiting in the depth of humility,
-and then I doubt not but you will be favoured with the presence of God,
+and then I doubt not that you will be favoured with the presence of God,
 both to your comfort and edification;
-for it is the Divine Monitor from heaven that is still the teacher of his people.
-It is He that yet speaks as never man spoke, and he cannot be removed into a corner.
-I hope dear cousin, you are sensible of these things,
+for it is the Divine Monitor from heaven that is still the teacher of His people.
+It is He that yet speaks as never man spoke, and He cannot be removed into a corner.
+I hope, dear cousin, you are sensible of these things,
 and I desire that you may increase in wisdom.
 Let it be your concern to keep a conscience void of offence both toward God and man.
 This is what I desire to hear of you and my other relatives.`"
 
+--
+
 [.small-break]
```

Comment:
This small break can be removed, because there is already a break in the text after the -- when the letter ends.